packages/playground/src/api/annotations/types.ts

```diff
@@ -1,17 +1,14 @@
 // Copyright 2022 Cognite AS
 
 import {
-  CogniteExternalId,
   CogniteInternalId,
   FilterQuery,
   IdEither,
   InternalId,
   SetField,
-  SinglePatch,
 } from '@cognite/sdk';
 
 export type AnnotatedResourceType = 'file';
```

Comment:
Looks like this is outdated, see https://github.com/cognitedata/context-api/blob/master/src/api/annotations_v2/schemas.py#L21L24

Reply:
Good point. I would prefer if we did this in a separate PR to keep this change as atomic as possible